docs/access_rights/organization_account.md

```diff
@@ -8,53 +8,54 @@ paths:
 
 # Organization account
 
-> **Warning** <br/>
-> The organization account is currently in the **closed beta testing** stage and available only to users with [**business**](https://apify.com/pricing) or higher subscriptions.<br/>
-> If you are interested in joining the closed beta testing, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+> The organization account is only available to accounts using our new billing system. If you see the migration message when you [log in](https://my.apify.com), you will need to migrate.
```

Comment:
"migration message" is not clear. Perhaps we could add screenshot and link the blogost about migration.